ui/src/components/common/Wallet.tsx

```diff
@@ -3,30 +3,34 @@ import { Button, Card, Header, Form } from 'semantic-ui-react'
 
 import { useParty, useLedger } from '@daml/react'
 import { useWellKnownParties } from '@daml/dabl-react'
+import { Broker } from '@daml.js/da-marketplace/lib/Marketplace/Broker'
 import { Investor } from '@daml.js/da-marketplace/lib/Marketplace/Investor'
+import { MarketRole } from '@daml.js/da-marketplace/lib/Marketplace/Utils'
 
 import { WalletIcon } from '../../icons/Icons'
 import { ExchangeInfo, DepositInfo, wrapDamlTuple } from '../common/damlTypes'
 import { parseError, ErrorMessage } from '../common/errorTypes'
 import FormErrorHandled from '../common/FormErrorHandled'
 import Page from '../common/Page'
 
+
 type Props = {
     deposits: DepositInfo[];
     exchanges: ExchangeInfo[];
+    role: MarketRole;
     sideNav: React.ReactElement;
     onLogout: () => void;
 }
 
-const InvestorWallet: React.FC<Props> = ({ deposits, exchanges, sideNav, onLogout }) => {
+const WalletC: React.FC<Props> = ({ deposits, exchanges, role, sideNav, onLogout }) => {
```

Comment:
What does the `C` stand for in the component name?

Reply:
There was a naming conflict if I let this as `Wallet`. So I changed it to `WalletC` (as wallet Component). I'll probably change the name to Holdings or something along those lines